src/components/MultiHopTrade/hooks/useTradeExecution/useTradeExecution.ts

```diff
@@ -124,17 +124,17 @@ export const useTradeExecution = ({
       chainId,
     })
 
-    setSellTxId(sellTxId)
+    setSellTxHash(sellTxId)
```

Comment:
Do we want to update `sellTxId` to `_sellTxhash` on l.121 above as well?

Reply:
resolved in https://github.com/shapeshift/web/pull/4884#discussion_r1252251787